release-team/release-team-removal.md

```diff
@@ -0,0 +1,52 @@
+- Failure to attend 3 consecutive release meetings without communication to the role lead
+- Failure to perform shadow tasks in due time, after 2-3 discussions with the role lead and the EA about performance
+- Unresponsive on Slack to messages for more than 2 days during non-peak time and 24 hours during peak time, without prior notice of absence. Peak time is different depending on the team's responsibility
+- Violation of [CNCF Code of Conduct](http://cncf.io/conduct/)
+
+Note that communications ahead of time to inform the leads or find alternative arrangements are greatly preferred in these circumstances.
+### Process
+It is important to be kind and patient while trying to resolve the situation.
+- Upon a pattern of inactivity, role lead will notify EA and have initial discussion with shadows about performance
+- If the shadow is still not contributing to the team, EA and lead will remove the shadow from the team, and find a replacement if appropriate. EA will be responsible for finding backfill.
+
+### Consequences
+- If the current release is the reason the shadow has Kubernetes membership, membership should be revoked
+- Removal of name in release-team document
+- Shadow will not be able to return to the following release. However, they are welcome to apply to releases after that
+
+## Role Lead Removal Guideline
+A role lead has been selected by your previous team lead to supervise, perform the duties of your team 
+and mentor your shadows to be future leads. Failure to perform lead duty can have a lasting impact on the release team.
+Reasons for removal could be including but not limited to:
+- Failure to attend or send appropriate representation to 2 consecutive release meetings
+- Failure to perform tasks expected of a role lead
+- Unresponsive on Slack to messages for more than 2 days during non-peak time and 24 hours during peak time, without prior notice of absence. Peak time is different depending on team
+- Violation of [CNCF Code of Conduct](http://cncf.io/conduct/)
+
+### Process
+- SIGs outside of SIG-release, shadows, leads, or contributors can express concern to release lead or EA.
+- Release lead and EA will have initial discussion with role lead
+- If the situation is not improved, the release lead and EA will have another discussion with role lead and begin to look for a replacement.
+- If deemed necessary by the release lead, EA and sig-release chairs, the role lead will be replaced with an experience shadow or someone who previously led the same role. EA will be responsible for finding backfill
+  
+### Consequences
+- Removal of name in release-team document
+- Role lead will need to shadow again and demonstrate sufficient responsibility, and leading ability, before holding another leadership position on the release team
+
+## Appeal
+If the release team member disagrees with their removal decision. They can escalate to SIG-Release chairs via private messaging on Slack.
```

Comment:
```suggestion
If the release team member disagrees with their removal decision, they can escalate to SIG Release chairs via private messaging on Slack or by emailing sig-release-leads@kubernetes.io.
```

Maybe emailing all of the SIG Release leads at once is better here? Versus potentially DMing only one of them.

Reply:
removed the DM option